### DISCUSSION TITLE
Counsel Timer, partially works

### DISCUSSION THREAD

**Discussion by moguerri85** [AntonyCorbett/OnlyT#463] [General]
there is a bug! this feature works only if there is an analog clock.
if you hide the analog clock via the split-screen, this functionality is not visible.

**Comment by AntonyCorbett** [comment]
@moguerri85 Thanks for the report. For the benefit of others, the bug relates to the **Automatic mode => Persist student time** and **Timer window => Persist talk duration** settings. If **Persist student time** is enabled, the student's time is displayed for a short pre-defined duration at the conclusion of the assignment. However, if you have used the **Split-screen width** option in the **Timer window** settings to completely hide the analogue clock, the student's time is not persisted.

**Comment by AntonyCorbett** [comment]
@moguerri85 Please see latest release for a fix.

**Reply by moguerri85** [reply]
thank u!
